### PR TITLE
Build and minor source refactoring

### PR DESCRIPTION
**Pull request by cchantep**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -13,37 +13,18 @@ libraryDependencies ++= Seq(
   "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.1" % Test
 )
 
+wartremoverErrors ++= Warts.unsafe
```
Comment (cchantep) [inline build.sbt]: Plugin was there, but not enabled ...

File: build.sbt [hunk 2]
```diff
@@ -13,37 +13,18 @@ libraryDependencies ++= Seq(
...
+
 scalacOptions ++= Seq(
   "-deprecation", // Emit warning and location for usages of deprecated APIs.
   "-encoding",
   "utf-8", // Specify character encoding used by source files.
   "-explaintypes", // Explain type errors in more detail.
   "-feature", // Emit warning and location for usages of features that should be imported explicitly.
-  "-language:existentials", // Existential types (besides wildcard types) can be written and inferred
```
Comment (cchantep) [inline build.sbt]: Better to import such feature (only) when required (more explicit)

File: build.sbt [hunk 3]
```diff
@@ -13,37 +13,18 @@ libraryDependencies ++= Seq(
...
-  "-language:experimental.macros", // Allow macro definition (besides implementation and application)
-  "-language:higherKinds", // Allow higher-kinded types
-  "-language:implicitConversions", // Allow definition of implicit functions called views
   "-unchecked", // Enable additional warnings where generated code depends on assumptions.
   "-Xcheckinit", // Wrap field accessors to throw an exception on uninitialized access.
   "-Xfatal-warnings", // Fail the compilation if there are any warnings.
-  "-Xfuture", // Turn on future language features.
```
Comment (cchantep) [inline build.sbt]: Avoid using beta features

File: build.sbt [hunk 4]
```diff
@@ -13,37 +13,18 @@ libraryDependencies ++= Seq(
...
-  "-Xlint:adapted-args", // Warn if an argument list is modified to match the receiver.
```
Comment (cchantep) [inline build.sbt]: Replaced by global `-Xlint`

File: build.sbt [hunk 5]
```diff
@@ -53,12 +34,7 @@ scalacOptions ++= Seq(
   "-Ywarn-nullary-override", // Warn when non-nullary `def f()' overrides nullary `def f'.
   "-Ywarn-nullary-unit", // Warn when nullary methods return Unit.
   "-Ywarn-numeric-widen", // Warn when numerics are widened.
-  "-Ywarn-unused:implicits", // Warn if an implicit parameter is unused.
```
Comment (cchantep) [inline build.sbt]: Replaced by global `-Ywarn-unused`

File: app/wiring/AppLoader.scala [hunk 6]
```diff
@@ -33,19 +34,18 @@ class AppComponents(context: Context)
...
-     Note the use of double(num) to bind only numbers (built-in :)
-     */
-    case GET(p"/sqrt/${double(num)}") =>
-      Action.async {
-        val query = Query(implicit c =>
-          SQL"select sqrt($num) as result".as(SqlParser.int("result").single))
-        queryRunner.run(query).map(r => Ok(r.toString))
-      }
+    case GET(p"/sqrt/${double(num)}") => Action.async {
+      /*
```
Comment (stankoua) [inline app/wiring/AppLoader.scala]: I think we can remove this comment

**Pull request merged**